The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/17548043488/job/49834012102) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/49834012102) [(plain)](https://github.com/rust-lang/rust/commit/be284b697fef4c560e040a433fbe2b832ed8426d/checks/49834012102/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
finished building tool typos
error: `capabilites` should be `capabilities`
##[error]  --> library/std_detect/src/detect/os/linux/aarch64.rs:213:21
    |
213 |         // Hardware capabilites from bits 32 to 63 should only
    |                     ^^^^^^^^^^^
    |
error: `capabilites` should be `capabilities`
##[error]  --> library/std_detect/src/detect/os/linux/aarch64.rs:216:27
    |
216 |         // these hardware capabilites will default to `false`.
    |                           ^^^^^^^^^^^
    |
tidy error: checks with external tool 'typos' failed
some tidy checks failed
Bootstrap failed while executing `test src/tools/tidy tidyselftest --extra-checks=py,cpp,js,spellcheck`
Command `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools-bin/rust-tidy /checkout /checkout/obj/build/x86_64-unknown-linux-gnu/stage0/bin/cargo /checkout/obj/build 4 /node/bin/npm --extra-checks=py,cpp,js,spellcheck` failed with exit code 1
Created at: src/bootstrap/src/core/build_steps/tool.rs:1550:23
Executed at: src/bootstrap/src/core/build_steps/test.rs:1281:29

Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:02:32
  local time: Mon Sep  8 10:46:11 UTC 2025
  network time: Mon, 08 Sep 2025 10:46:11 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/std_detect/src/detect/os/linux/aarch64.rs
+++ b/library/std_detect/src/detect/os/linux/aarch64.rs
@@ -210,10 +210,10 @@ impl From<auxvec::AuxVec> for AtHwcap {
             ..Default::default()
         };
 
-        // Hardware capabilites from bits 32 to 63 should only
+        // Hardware capabilities from bits 32 to 63 should only
         // be tested on LP64 targets with 64 bits `usize`.
         // On ILP32 targets like `aarch64-unknown-linux-gun_ilp32`,
-        // these hardware capabilites will default to `false`.
+        // these hardware capabilities will default to `false`.
         // https://github.com/rust-lang/rust/issues/146230
         #[cfg(target_pointer_width = "64")]
         {

--- a/library/std_detect/src/detect/os/linux/aarch64.rs
+++ b/library/std_detect/src/detect/os/linux/aarch64.rs
@@ -212,7 +212,7 @@ impl From<auxvec::AuxVec> for AtHwcap {
 
         // Hardware capabilities from bits 32 to 63 should only
         // be tested on LP64 targets with 64 bits `usize`.
-        // On ILP32 targets like `aarch64-unknown-linux-gun_ilp32`,
+        // On ILP32 targets like `aarch64-unknown-linux-gnu_ilp32`,
         // these hardware capabilities will default to `false`.
         // https://github.com/rust-lang/rust/issues/146230
         #[cfg(target_pointer_width = "64")]

